library/core/src/cmp.rs

```diff
@@ -3,14 +3,17 @@
 //! This module contains various tools for comparing and ordering values. In
 //! summary:
 //!
-//! * [`Eq`] and [`PartialEq`] are traits that allow you to define total and
-//!   partial equality between values, respectively. Implementing them overloads
-//!   the `==` and `!=` operators.
+//! * [`Eq`] and [`PartialEq`] correspond to equivalence relations and partial
+//!   equivalence relations, respectively, and they overload the `==` and `!=`
+//!   operators.
```

Comment:
```suggestion
//!   equivalence relations, respectively. Implementing them provides the `==` and `!=`
//!   operators for types.
```

Reply:
I have stumbled upon the wording "overloading" too, but I noticed it's also used for `Ord` and `PartialOrd`. Should I remove the term "overloading" completely?

Reply:
@JanBeh Not in this PR, but since you're changing *this* text anyway, I think we should take the opportunity to make this clearer.

Reply:
Done, but notice that "overloading an operator" is also used in the same list (for `<`, `<=`, `>`, and `>=`). If you want I can fix it as well, or leave as is.